dotcms-integration/src/test/java/com/dotcms/MainSuite2b.java

```diff
@@ -317,7 +318,8 @@
         Task240530AddDotAIPortletToLayoutTest.class,
         EmbeddingContentListenerTest.class,
         Task240606AddVariableColumnToWorkflowTest.class,
-        OpenAIContentPromptActionletTest.class
+        OpenAIContentPromptActionletTest.class,
+        JobQueueManagerAPITest.class
```

Comment:
I suggest to move the test to another test suite because MainSuite2b is already executing many tests. These is the time it takes to execute per suite: 

- JVM IT Tests MainSuite 1a (pull_request) Successful in 34m
- JVM IT Tests MainSuite 1b (pull_request) Successful in 29m
- JVM IT Tests MainSuite 2a (pull_request) Successful in 59m
- JVM IT Tests MainSuite 2b (pull_request) Successful in 24m

Reply:
You're right! Forget my comment!